ssh_ldap_pubkey/__init__.py

```diff
@@ -84,6 +84,9 @@ def connect(self):
             conn.network_timeout = conf.bind_timeout
             conn.timeout = conf.search_timeout
 
+            if conf.ssl == 'start_tls' and conf.uri.startswith('ldap://'):
```

Comment:
I noticed that it'd throw an exception if you tried to do StartTLS if you connected via an ldaps:// uri, presumably because communication is already encrypted. Although I suppose if you've specified an ldaps:// uri AND specified `ssl start_tls` in the config file it could be argued that it's a misconfiguration and *should* fail?